core/storage/job/gae_models.py

```diff
@@ -86,6 +93,17 @@ def is_cancelable(self):
 
     @classmethod
     def get_recent_jobs(cls, limit, recency_msec):
+        """Gets at most limit jobs with respect to a time after recency_msec.
+
+        Args:
+            limit: int. A limit on the number of jobs to return.
+            recency_msec: int. A time in milliseconds. After this time a
+                job is considered recent.
+
+        Returns:
+            list(Jobs) or None. A list of at most limit jobs
```

Comment:
So I ended up finding a place in core/controllers/admin.py where I can trace my method back and I used loggin.info.(type(...)). I got bellow output, but I'm not sure how to interpret it. Is the type JobModel?

INFO     2017-03-18 02:47:06,930 jobs.py:1093] JobModel<additional_job_params=JsonProperty('additional_job_params'), created_on=DateTimeProperty('created_on', auto_now_add=True), deleted=BooleanProperty('deleted', default=False), error=TextProperty('error'), has_been_cleaned_up=BooleanProperty('has_been_cleaned_up', default=False), job_type=StringProperty('job_type'), last_updated=DateTimeProperty('last_updated', auto_now=True), metadata=JsonProperty('metadata'), output=JsonProperty('output'), status_code=StringProperty('status_code', default='new', choices=frozenset(['new', 'started', 'failed', 'completed', 'queued', 'canceled'])), time_finished_msec=FloatProperty('time_finished_msec'), time_queued_msec=FloatProperty('time_queued_msec'), time_started_msec=FloatProperty('time_started_msec')>